packages/forms/src/UIForm/utils/properties.test.js

```diff
@@ -1,7 +1,7 @@
-import { getValue, omit } from './properties';
+import { getValue, omit, convertValue } from './properties';
 
 describe('Properties utils', () => {
-	describe('getValue', () => {
+	describe('#getValue', () => {
```

Comment:
So in the description we can see `Properties utils #getValue should ...` 
It's a way to mark/highlight the function that is tested